### PR TITLE
Bjorncs/athenz ssl context

### PR DESCRIPTION
**Pull request by bjorncs**

### CODE REVIEW

**Comment by bjorncs** [comment]
Added a missing initalization of `KeyStore` in `SSLContext` builder which was the cause of the failing build last night.

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/athenz/impl/AthenzSslContextProviderImpl.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    private static KeyManager[] createKeyManagersWithServiceCertificate(ZtsClient ztsClient) {
+        try {
+            AthenzIdentityCertificate identityCertificate = ztsClient.getIdentityCertificate();
+            KeyStore keyStore = KeyStore.getInstance("JKS");
+            keyStore.load(null);
+            keyStore.setKeyEntry("athenz-controller-key",
+                                 identityCertificate.getPrivateKey(),
+                                 new char[0],
+                                 new Certificate[]{identityCertificate.getCertificate()});
+            KeyManagerFactory keyManagerFactory = KeyManagerFactory.getInstance("X509");
```
Comment (tokle) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/athenz/impl/AthenzSslContextProviderImpl.java]: X509 is not supported, use SunX509 instead

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/athenz/impl/AthenzSslContextProviderImpl.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        }
+    }
+
+    private static TrustManager[] createTrustManagersWithAthenzCa(AthenzConfig config) {
+        try {
+            KeyStore trustStore = KeyStore.getInstance("JKS");
+            try (FileInputStream in = new FileInputStream(config.athenzCaTrustStore())) {
+                trustStore.load(in, "changeit".toCharArray());
+            }
+            TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance("X509");
```
Comment (tokle) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/athenz/impl/AthenzSslContextProviderImpl.java]: SunX509

**Review by tokle — CHANGES_REQUESTED** [review]

**Comment by bjorncs** [comment]
@tokle PTAL

**Review by tokle — APPROVED** [review]

**Pull request merged**